src/prefect/orion/schemas/core.py

```diff
@@ -364,6 +364,7 @@ class TaskRun(ORMBaseModel):
     )
     cache_key: Optional[str] = Field(
         default=None,
+        max_length=2_000,
```

Comment:
Can you move this to a setting so OSS servers can change it if they want and add it to the tasks concept documentation?